### ISSUE TITLE
upgrade from 5.2.1 to 6.0.2 / video cannot play

### ISSUE THREAD

**Issue by tomtoom1978** [YunoHost-Apps/peertube_ynh#405]
- Server: computer 2020
- YunoHost version: 11.2.9.1

After the successful update of peertube, all the videos inside the instance cannot play anymore.
The interface, log, picture, etc.. working well, it's even possible to download the video (with download fonction), but not to play inside the player.

i can see:
- the videos files are present inside the storage
- the database working
- **the static link not working**
- but the download link working
- upload a new video, nothing changed
- restart HLS video transcode, nothing changed

i'll back here soon, with more information

**Comment by lapineige** [comment]
#404 might solve it ?

**Comment by ericgaspar** [comment]
please test
```
sudo yunohost app upgrade peertube -u https://github.com/YunoHost-Apps/peertube_ynh/tree/testing --force
```

**Comment by ericgaspar** [comment]
what is not working? a little more context will definitely help.

**Comment by lapineige** [comment]
Video does not play, nor load as the duration is not updated in the player footer.
Tested with old and new videos.

**Comment by tomtoom1978** [comment]
> #404 might solve it ?

I just put back my backup, too much activity on my instance to test now...
sorry,

But we have the same pb:
`Peertube web log gives this Client log: HLS.js error: networkError - fatal: true - manifestLoadError`

**Comment by lapineige** [comment]
"Related" upstream issues ? https://github.com/Chocobozzz/PeerTube/issues?q=is%3Aissue+HLS.js+error%3A+networkError+-+fatal%3A+true+-+manifestLoadError

**Comment by lapineige** [comment]
Update reverted, new change available here : #408

Probably not working for now.

**Comment by ericgaspar** [comment]
issue fixed 🥶 (6.0.2 needs more work and it is back to test)
